datahost-ld-openapi/src/tpximpact/datahost/ldapi/handlers.clj

```diff
@@ -61,9 +60,29 @@
   (if-let [_release (db/get-release db series-slug release-slug)]
     (let [api-params (get-api-params request)
           incoming-jsonld-doc body-params
-          {:keys [_op jsonld-doc]} (db/insert-revision! db api-params incoming-jsonld-doc)]
+          {:keys [_op jsonld-doc resource-id]} (db/insert-revision! db api-params incoming-jsonld-doc)]
       {:status 201
+       :headers {"Location" (str "/data/" series-slug "/release/" release-slug "/revisions/" resource-id)}
        :body jsonld-doc})
 
     {:status 422
-     :body "Release for this revision does not exist"}))
\ No newline at end of file
+     :body "Release for this revision does not exist"}))
+
+;; TODO: Uploaded change data should be validated against a supplied schema
+(defn post-revision-changes [db {{:keys [series-slug release-slug revision-id]} :path-params
+                                 {{:keys [appends]} :multipart} :parameters
+                                 body-params :body-params :as request}]
+  (if-let [_revision (db/get-revision db series-slug release-slug revision-id)]
+    (let [incoming-jsonld-doc body-params
+          api-params (-> (get-api-params request)
+                         ;; TODO: coercion doesn't seem to be working on revision-id for some reason
```

Comment:
@kiramclean I don't know if you have any pointers as to why `revision-id` isn't being coerced to an int in the `api-params`. It's specified as `int?` in the parameters map.